### DISCUSSION TITLE
Next release date

### DISCUSSION THREAD

**Discussion by enys** [argoproj-labs/applicationset#383] [General]
A few nice features have been merged to master, including  #366, any idea of a rough date for the next release ?

Thanks,

**Comment by jgwest** [comment]
We should be releasing alongside the next major version of Argo CD (v2.2), whose ETA is roughly 2-3 weeks from now.

**Reply by enys** [reply]
Thanks ! Any idea where I could have found out that date ? I looked here : https://github.com/argoproj/argo-cd/milestone/23

**Reply by jgwest** [reply]
IIRC I heard it on Argo CD contributor meeting (or was it the Argo CD/Rollouts meeting?), both of which are public Zoom meetings (you can find details in the Argo project calendar).

**Reply by enys** [reply]
thanks !
